sklearn/metrics/_plot/roc_curve.py

```diff
@@ -145,8 +148,14 @@ def plot(
             "linestyle": "--",
         }
 
-        if chance_level_kw is not None:
-            chance_level_line_kw.update(**chance_level_kw)
+        if chance_level_kw is None:
```

Comment:
Let's make the cod a bit consistent and rename `chance_level_line_kw` as `default_chance_level_line_kw` and then pass call the function and pass the argument by position.

Since we make copy of the dictionary internally, we will be safe. We only need to pass the output of the function to the plotting function.